tox.ini

```diff
@@ -16,7 +12,7 @@ envlist =
 whitelist_externals =
     pytest
 commands =
-    pytest tests {posargs} \
+    pytest {posargs:tests} \
```

Comment:
what the use of posargs?

Reply:
This allows you to pass on arguments when running `tox`, e.g.
```bash
tox -e py tests/physics
```
Here, `tests/physics` is the `posargs`.

Reply:
I see, so its either posargs or if not given then tests by default?

Reply:
Yep, it's the same as in bash scripts or functions (e.g. `${0:-default argument}`)

Reply:
Ok, did not know that. Thanks